plugins/org.eclipse.n4js.ui/src/org/eclipse/n4js/ui/contentassist/CustomN4JSParser.java

```diff
@@ -108,6 +109,22 @@ protected CustomInternalN4JSParser createParser() {
 		return result;
 	}
 
+	/**
+	 * Workaround for Xpect which messes with the injectors and singletons.
+	 *
+	 * Rather than trying to find the rule name in the cached instance of ruleNames, we invoke the entry rule directly.
+	 */
+	@Override
+	protected Collection<FollowElement> getFollowElements(AbstractInternalContentAssistParser parser) {
```

Comment:
I don't see any difference to the overwritten method `N4JSParser#getFollowElements(AbstractInternalContentAssistParser)`. - What am I overlooking?

Reply:
We used to generate this in a custom mwe2 generator fragment which made use of reflection. I could remove the custom generator fragment. Originally it was a workaround for a bug in Xtext. The bug was fixed in the meantime and therefore we can now use standard means and override `getFollowElements`.